### PR TITLE
Events Page

### PR DESCRIPTION
**Pull request by esarajyan**
This PR adds the events page.

image
image
image
image
image

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            <li><a class="navModal--link" href="/events.html">Events</a></li>
+            <li><a class="navModal--link" href="/services.html">Services</a></li>
+            <li><a class="navModal--link" href="#">Shop</a></li>
+            <li><a class="navModal--link" href="/get_involved.html">Get Involved</a></li>
+            <li><a class="btnLink btnLink-large">Get Updates</a></li>
+            <li><a class="btnLink btnLink-red btnLink-large">Donate</a></li>
+        </ul>
+    </div>
+
+    <p class="event--title l-center">Events</p>
```
Comment (Tara-Tiong) [inline events.html]: I think you should change this to `<h2 class="sectionHeading sectionHeading-spaced .sectionHeading-centered">`

File: events.html [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+    <div class="l-center">
+        <iframe src="https://calendar.google.com/calendar/embed?height=600&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=America%2FLos_Angeles&amp;src=aWQ5ZmozZ2NnZzZsaGhhb2hxbGk3YTkxamNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&amp;color=%238E24AA&amp;showTitle=1&amp;showNav=0&amp;showDate=0&amp;showPrint=0&amp;showTabs=0&amp;showTz=0&amp;showCalendars=1" style="border:solid 1px #777" width="800" height="600" frameborder="0" scrolling="no"></iframe>
+    </div>
+
+    <div class="l-section">
```
Comment (Tara-Tiong) [inline events.html]: use section tags `<section class="">`

File: events.html [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                    </div>
+                    <div class="event--btn2">
+                        <a class="btnLink btnLink-small">Learn More</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <p class="sectionHeading l-center l-section">Past events...</p>
```
Comment (Tara-Tiong) [inline events.html]: - use `<h2 class="sectionHeading sectionHeading-spaced .sectionHeading-centered">` instead 
- and add  the tag `<section class="l-section">` around line 131-161

File: events.html [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+            <div class="infoCard">
+                <div class="infoCard--caption">
+                    <p class="infoCard--title">Classes</p>
+                    <p class="infoCard--text">Due to concerns over the outbreak of coronavirus (COVID-19) we have decided to postpone in person events and classes. Sign up for our email list to stay up to date with BAMP.</p>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="gridBox gridBox-col2 gridBox-center l-section">
```
Comment (Tara-Tiong) [inline events.html]: remove `l-section` and add `<section class="l-section">` around lines 60-129

File: events.html [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+            <a class="header--link" href="/programs.html">Programs</a>
+            <a class="header--link" href="/events.html">Events</a>
+            <a class="header--link" href="/services.html">Services</a>
+            <a class="header--link" href="#">Shop</a>
+            <a class="header--link" href="/get_involved.html">Get Involved</a>
+            <a class="header--link header--link-tight" href="#"><span class="btnLink btnLink-small">Get Updates</span></a>
+            <a class="header--link header--link-tight" href="#"><span class="btnLink btnLink-red btnLink-small">Donate</span></a>
+            <button id="hamburger" class="header--hamburger">☰</button>
+        </div>
+    </body>
```
Comment (Tara-Tiong) [inline events.html]: I think the `</body>` tag should be the second to last (right above `</html>`) of the whole page

File: events.html [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+        <div class="fixedRatio fixedRatio-60">
+            <img class="fixedRatio--contents" src="/images/events/FAF4A214-4C57-40D5-A6B5-4C71BD66BE8B.JPG">
+        </div>
+    </div>
+
+    <div class="footer">
+        <div class="footer--panel">
+            <div class="footer--bizCard">
+                <div class="footer--mobileBizCard">
+                    <img src="../images/logo-gold-border.svg">
```
Comment (Tara-Tiong) [inline events.html]: add ` <a href="/index.html"></a>`

File: events.html [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+                    <div class="footer--mobileButton">
+                        <a class="btnLink" href="#">Contact</a><br><br>
```
Comment (Tara-Tiong) [inline events.html]: add `mailto: bampteamad@gmail.com` instead of `#`

File: events.html [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+            <li><a class="navModal--link" href="#">Shop</a></li>
+            <li><a class="navModal--link" href="/get_involved.html">Get Involved</a></li>
+            <li><a class="btnLink btnLink-large">Get Updates</a></li>
+            <li><a class="btnLink btnLink-red btnLink-large">Donate</a></li>
+        </ul>
+    </div>
+
+    <p class="event--title l-center">Events</p>
+
+    <div class="l-center">
```
Comment (Tara-Tiong) [inline events.html]: use a section tag

File: events.html [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+                        <a class="btnLink btnLink-small">Learn More</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <p class="sectionHeading l-center l-section">Past events...</p>
+
+    <div class="gridBox gridBox-col3 gridBox-center l-section">
```
Comment (Tara-Tiong) [inline events.html]: remove `l-section`

File: events.html (hunk 1 above)
Comment (johncip) [reply events.html]: agreed, but no dot before the last one

**Review by johncip — CHANGES_REQUESTED** [review]
agree with Tara's remarks -- let me know when those changes are in & I'll review again

**Review by johncip — APPROVED** [review]
nice work!

**Pull request merged**